java/mtsj/core/src/main/resources/db/specific/h2/V0001__R001_Create_schema.sql

```diff
@@ -97,7 +97,7 @@ CREATE TABLE Image (
   id BIGINT NOT NULL AUTO_INCREMENT,
   modificationCounter INTEGER NOT NULL,
   name VARCHAR(255),
-  content VARCHAR(2147483647),
+  content clob,
```

Comment:
should be capitalized (CLOB) to be homogenous...

Reply:
will be done after merge

Reply:
@vapadwal please create and issue and/or PR for the leftovers and tidy up. Great, that we have finally merged this after a long way now...